debian/control

```diff
@@ -31,7 +31,6 @@ Pre-Depends: dpkg (>= 1.16.1)
 Depends:
  adduser,
  debconf,
- python3-distutils|libpython3-stdlib (<< 3.6),
```

Comment:
nb that `python3.X` depends on `libpython3.X-stdlib` so we don't need to pull it in explicitly.

This was here to say that we didn't need a separate `python3-distutils` provided `libpython3-stdlib` was old enough. Now we don't need a separate python3-distutils at all, so the whole line is redundant.